packages/registry/docs/conceptual-guides/add-on-styles-loader.md

```diff
@@ -0,0 +1,43 @@
+---
+myst:
+  html_meta:
+    "description": "An explanation of the add-ons styles loader in @plone/registry"
+    "property=og:description": "An explanation of the add-ons styles loader in @plone/registry"
+    "property=og:title": "Add-ons styles loader"
+    "keywords": "@plone/registry, registry, add-ons, loader"
+---
+
+# Add-ons styles loader
+
+The `@plone/registry` compatible add-ons are able to define styles that should be loaded by the app.
+This is achieved by creating a folder {file}`styles` with a {file}`_main.css` file that serves as entry point.
```

Comment:
Why the underscore? It makes it look unimportant when actually it's especially important.

Reply:
I agree. Let's remove the `_`. `main.css` makes sense too?

Reply:
`_` sorts earlier in a directory listing, but that's not important to naming things well. I often see `_` in front of SCSS files, denoting it as a [partial](https://sass-lang.com/guide/#topic-4). There ain't nothing partial about what's in `_main.css`. I also don't want to hold the shift key to get an underscore when typing docs to mention this file. I'd go with `main.css`.